node/nodeRunner.go

```diff
@@ -875,8 +867,8 @@ func (nr *nodeRunner) CreateManagedConsensusComponents(
 		ScheduledProcessor:    scheduledProcessor,
 		IsInImportMode:        nr.configs.ImportDbConfig.IsImportDBMode,
 		ShouldDisableWatchdog: nr.configs.FlagsConfig.DisableConsensusWatchdog,
-		ConsensusModel:        nr.getConsensusModelFunc(),
-		ChainRunType:          nr.getChainRunTypeFunc(),
+		ConsensusModel:        nr.getConsensusModel(),
+		ChainRunType:          nr.getChainRunType(),
```

Comment:
we can remove the function `getChainRunType` and always use `common.ChainRunTypeRegular` in node/nodeRunner.go as this responsible with the regular node assembly. Valid also for the `getConsensusModel` function

Reply:
Right, good suggestion. Applied it